_docs/concepts/policy-and-control/mixer.md

```diff
@@ -86,9 +80,7 @@ Refer [here](./mixer-config.html) for detailed information on Mixer's configurat
 
 When a request comes in to Mixer, it goes through a number of distinct handling phases:
 
-- **Supplementary Attribute Production**. The first thing that happens in Mixer is to run a globally configured
-set of adapters that are responsible for introducing new attributes. These attributes are combined with the attributes
-from the request to form the total set of attributes for the operation.
+- **Supplementary Attribute Production**. The first thing that happens in Mixer is to run a globally configured set of adapters that are responsible for introducing new attributes. These attributes are combined with the attributes from the request to form the total set of attributes for the operation.
```

Comment:
the Mixer

Reply:
Better phrased as "The Mixer initially runs a globally configured set of adapters that are responsible for introducing new attributes".

Reply:
Updated.